allow selection to use a3 markers as imageUrls for point markers

indig0fox opened the pull request:
Markers have been brought in, and a placeholder will auto-open for marker selection when you load the page. This is a rough sketch and idea of choosing a marker by the addon its from, as some will have overlapping names.

Need some help integrating this somehow into the existing prompt set or a picker similar to 'color'. Once better put together, I can implement the marker recoloring to emulate in-game color variants.

indig0fox committed:
import markers, add routes, placeholder selection

MnkyArts commented:
We could probably add a window/prompt when placing a marker which has a dropdown or an area with all of the a3 markers

//Edit: just saw you already did something like this, we could integrate the color selection directly into the selection prompt. Need to look at your selection system to understand it a bit more

indig0fox committed:
implement sequelize ORM

indig0fox committed:
many changes, see message

- moves sequelize DB stuff and express routes to /lib commonJS modules
- adds env use to docker-compose and node.js
- node server will now connect to the DB, create the .env database if it doesn't exist, then populate it with the appropriate tables etc if not already set up
- node server will also (wipe and) populate the markers found in /public/markers into the DB for api return to frontend
- implements PostGIS (spatially-aware PostgresDB) in docker compose w persistent volume la3map_pg_data
- docker compose file has been updated to support healthchecks for both services & app dependency on DB running
- docker compose will send env variable IS_DOCKER if running via docker to adjust for service hostname rather than assuming the service is running on localhost, or whatever is provided as POSTGRES_HOST in .env
- saves marker geo data as geometry columns instead of raw json
